index.qmd

```diff
@@ -53,18 +59,26 @@ The above commands will give you the current versions of the packages used in
 this book. If you're having trouble reproducing some of the results, it may be
 due to package updates that took place after the book was last updated. You can
 try matching the package versions we used to build the book by one of the
-following methods. You may be required to obtain a [GitHub Personal Access
-Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
-and store it in the environment variable `"GITHUB_PAT"`.
+following methods. You will need a [GitHub Personal Access Token
+(PAT)](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
+and store it in the environment variable `"GITHUB_PAT"`; a "classic" token with
+at least `public_repo` access should work.
 
 #### A: Download and use the `renv.lock`
 
 ```{r, eval = FALSE}
 # Run this once:
-install.packages("renv")
+install.packages(c("renv", "usethis", "gitcreds"))
```

Comment:
A prerequisite for this entire section is having a GitHub account. Is there another way that doesn't require this? Or just (B)?

Reply:
Another potential alternative: investigate pak lockfiles.  These are different from renv lockfiles and are probably less convenient for us to use, but might work without a user `GITHUB_PAT`.